python/xorq/flight/__init__.py

```diff
@@ -45,16 +60,17 @@ def port_in_use(self):
 
     @classmethod
     def from_defaults(cls, **kwargs):
-        (scheme, *_) = cls._constraints.allowed_schemes
-        _kwargs = toolz.merge(
+        (scheme, *_) = allowed_schemes
```

Comment:
since all fields have a default, we can do this
```
_kwargs = {
    **{attr.name: attr.default for attr in cls.__attrs_attrs__},
    **kwargs,
}
```

Reply:
on second thought, now that we have defaults, we don't need a `from_defaults` method, we can use the constructor itself
but i'd be happy to leave this as is because we'll likely need another PR to randomly choose a port